### ISSUE TITLE
How to upload a BIN file to ESP8266

### ISSUE THREAD

**Issue by pepperm2** [Qrome/marquee-scroller#206]
Well, the firmware update feature on one of my Marquee-Scroller's has failed and try as I might, I just cant get the code from git hub to compile without many errors. This I suspect is due to slightly incompatible libraries these days.

So, I figured... Just upload the BIN file. To that end I have been trying the ESP8266Flasher with the githum binary and cant get anything but the Wemos D1 blue LED to flash.

Are there any suggestions as how I can get the latest arduino code to compile ot how to upload the BIN file please?

Mark

**Comment by pepperm2** [comment]
> Change to the library files that are listed in the Readme.md -- it will compile. Usually it is the ESP8266 Core version.
> 
> Did you try doing the update with the binary file and the web interface? ![image](https://user-images.githubusercontent.com/32021227/138290631-7e57e32e-ac9b-4b9a-99e4-eb2ded23bdac.png)

Thanks for the reply...

Yes I tried the Firmware Update and the ESP8266 just doesnt do anything useful now. No display (well random dots) and no WIFI to connect to and to configure.

I have tried removing all the libraries and re-installing them from the sources detailed and still nothing compiles. I get errors like the following:
error: call to 'HTTPClient::begin' declared with attribute error: obsolete API, use ::begin(WiFiClient, url)

I have tried both the normal and Wide BINs, I just have 4 LED matrixes.
